src/main/java/org/microshed/lsp4ij/operations/completion/LSPCompletionContributor.java

```diff
@@ -127,9 +127,11 @@ private void addCompletionItems(PsiFile file, Editor editor, CompletionPrefix co
             // Create lookup item
             var lookupItem = createLookupItem(file, editor, completionPrefix.getCompletionOffset(), item, itemDefaults, languageServer);
             // Group it by using completion item kind
-            var groupedLookupItem = PrioritizedLookupElement.withGrouping(lookupItem, item.getKind().getValue());
+            int group = item.getKind() == null ? 0 : item.getKind().getValue();
```

Comment:
So this avoids a `NullPointerException` but what does `0` mean here? When I look at https://github.com/eclipse-lsp4j/lsp4j/blob/main/org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/CompletionItemKind.java all of the values returned by the `enum` are from 1 to 25.

Reply:
Good question. It gets put in a field in a new object and who knows how that is interpreted? I was thinking "unknown" but would Text(1) be a better default? I assume that is a pretty generic category.

Reply:
It was just an observation, but seemed like the default should be one of the defined values. It was the first time I'd looked at any of this and not familiar with this code so your guess is as good as mine. Assuming that it should be one of the values from the enum, I would suggest referencing the constant instead of the integer (e.g. `Text.getValue()` instead of `1`).

Reply:
I suggest that you see our LSP4IJ, we remove this code since sort should be never done by grouping item with kind but by using sortText.

Reply:
Ok, this point will be moot in the next update.